content/get-haiku/release-notes-r1b2.md

```diff
@@ -0,0 +1,271 @@
+
+### Installation Improvements
+
+There are various improvements in the installation process. The first is that there is better support for EFI, though work remains to be done. Additionally, the Installer now supports optional packages that are included on the Installation medium. 
+
+DriveSetup will now display more information about the drives in your system. It will show the used space of existing partitions, and it is now also able to identify encrypted volumes.
+
+Finally, there is now also an upgrade path available from Haiku R1 beta 1 to beta 2.
+
+<img src="/files/get-haiku/installer_optionalpackages.png"/><br>
+<small><i>Optional package selection in the Installer.</i></small>
+
+### The Terminal now has an emulated Meta-key
+
+Certain early workstation keyboards included a key labeled "Meta" that functioned as an extra modifier key alongside Shift and Control. Its purpose was to give users an easy way of entering special characters: When held down it caused the eighth bit to be set on all characters read from the keyboard, allowing access to ​the "extended" portion of the ASCII character set (character codes above 127). 
+
+Although the Meta key is absent from modern keyboards, two major pieces of UNIX software still rely on it being present: GNU Emacs and the GNU readline library, which is used by bash and a wide variety of other software to read input from the terminal. (Python's interactive shell uses readline, for instance.) These applications use the Meta key as a "command" key and provide additional editing features when it's available. 
+
+In beta 2, the Terminal will have functionality like macOS': It adds a configuration option to the "Settings..." dialog that, when enabled, causes the left Option key (only) to function as a Meta key. The right Option key retains its normal function, and can be used to enter special characters at the keyboard. 
+
+<img src="/files/get-haiku/terminal_metakey.png"/><br>
+<small><i>Enabling the meta key functionality in the settings.</i></small>
+
+## Other Enhancements
+
+### Applications
+**Deskbar**
+ - [#4971](https://dev.haiku-os.org/ticket/4971) Resize Deskbar from the Deskbar settings, our by using the CTRL-ALT-right click drag
+ - [#8537](https://dev.haiku-os.org/ticket/8537) In menus, the size of the icon will now always be the same size as your chosen font size
+ - [#8691](https://dev.haiku-os.org/ticket/8691) The icons of replicants are running in the Deskbar, will scale according to the settings
+ - [#13304](https://dev.haiku-os.org/ticket/13304) Auto-raised deskbar will now also auto-unraise
+
+**DriveSetup**
+ - [#10098](https://dev.haiku-os.org/ticket/10098) Encrypted disks will actually display that they are encrypted instead of empty
+
+**HaikuDepot**
```

Comment:
Doesn't all the work on HaikuDepot deserve a section in the "highlights"? To me it seems much more important than the meta key, for example. Well, I guess having an highlight section can only lead to bikeshedding about what goes there.